examples/airflow/README.md

```diff
@@ -1,43 +1,272 @@
+
+  ```bash
+  MARQUEZ_BACKEND=http             # Collect metadata using HTTP backend
+  MARQUEZ_URL=http://marquez:5000  # The URL of the HTTP backend
+  MARQUEZ_NAMESPACE=example        # The namespace associated with the collected metadata
+  ```
+  > **Note:** The `marquez.env` config file will be used by the `airflow`, `airflow_scheduler`, and `airflow_worker` containers to send lineage metadata to Marquez.
+  
+* Your `examples/airflow/` directory should now contain the following:
+
+  ```
+  .
+  ├── docker/
+  ├── docs/
+  ├── marquez.env
+  └── requirements.txt
+  ```
+
+## Step 2: Write Airflow DAGs using Marquez
+
+First, let's create the `dags/` folder where our example DAGs will be located:
 
 ```bash
-$ ./docker/up.sh
+$ mkdir dags
+```
+
+When writing our DAGs, we'll use [`marquez-airflow`](https://pypi.org/project/marquez-airflow) enabling Marquez to observe the DAG and automatically collect task-level metadata. Below, in steps `2.1` and `2.2`, we create the DAGs `counter` and `sum` and add them to `dags/`. You'll notice that to begin collecting DAG metadata, we only make the following change:
+
+```diff
+- from airflow import DAG
++ from marquez_airflow import DAG
 ```
 
-> **Tip:** Use the `--pull` to pull a tagged image.
+## Step 2.1: Create DAG `counter`
 
-Airflow listens on port `8080`. To view the Airflow UI and verify it's running, open http://localhost:8080. Then, browse to http://localhost:3000 to begin exploring DAG metadata via the Marquez UI.
+Under `dags/`, create a file named `counter.py` and copy in the following code:
 
-# Airflow
+```python
+import random
 
-[Airflow](https://airflow.apache.org) is a platform to programmatically author, schedule, and monitor workflows. Below we define some general concepts, but we encourage you to review [Airflow's concept doc](https://airflow.apache.org/docs/stable/concepts.html). 
+from marquez_airflow import DAG
+from airflow.operators.postgres_operator import PostgresOperator
+from airflow.utils.dates import days_ago
 
-### DAGs
+default_args = {
+    'owner': 'datascience',
+    'depends_on_past': False,
+    'start_date': days_ago(1),
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'email': ['datascience@example.com']
+}
 
-A `DAG` (or [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) is a collection of `Tasks`. It represents an entire workflow by describing the relationships and dependencies between tasks.
+dag = DAG(
+    'counter',
+    schedule_interval='*/1 * * * *',
+    catchup=False,
+    is_paused_upon_creation=False,
+    default_args=default_args,
+    description='DAG that generates a new count value between 1-10.'
+)
 
-### Tasks
+t1 = PostgresOperator(
+    task_id='if_not_exists',
+    postgres_conn_id='example_db',
+    sql='''
+    CREATE TABLE IF NOT EXISTS counts (
+      value INTEGER
+    );''',
+    dag=dag
+)
```

Comment:
A bash operator example would be a lower barrier to entry. It doesn't show off as much but it gets them running quickly.

Reply:
I agree, we can have a simple example just to get started, but breakout examples based on use cases